README.md

```diff
@@ -202,22 +202,26 @@ pop call contract -p ./my_contract --contract $INSTANTIATED_CONTRACT_ADDRESS --m
 For end-to-end testing you will need to have a Substrate node with `pallet contracts`.
 You do not need to run it in the background since the node is started for each test independently.
 To install the latest version:
-
 ```
 cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git
 ```
 
-If you want to run any other node with `pallet-contracts` you need to change `CONTRACTS_NODE` environment variable:
+Run e2e testing on the Smart Contract:
 
 ```sh
-export CONTRACTS_NODE="YOUR_CONTRACTS_NODE_PATH"
+# Run e2e tests for an existing smart contract
```

Comment:
Don't see the use for this comment, more cases like this in the README. Most of them have a description already above the code import. Should be fixed when fixing #183